scripts/apps/authoring/authoring/index.ts

```diff
@@ -340,7 +343,68 @@ angular.module('superdesk.apps.authoring', [
                         return api.find('archive', $route.current.params._id);
                     }],
                 },
-            });
+            })
+            .activity('unpulish', {
+                label: gettext('Unpublish'),
+                priority: 50,
+                icon: 'kill',
+                group: 'corrections',
+                controller: ['data', 'authoring', 'familyService',
+                    (data, authoring, familyService) => {
+                        const item = data.item;
+                        let relatedItems = [];
+
+                        familyService.fetchItems(item.archive_item.family_id, item)
```

Comment:
I would rather keep that unpublish endpoint consistent with other publishing actions for now,
it would make sense to have `/unpublish` with list of ids, it's not really important if an item is related or not, and the same would work for publishing from lists etc. but that's another task to refactor it all.
will check that related items endpoint, moving that logic with family id to server would be good for sure